data-raw/process_data_set_lizette.R

```diff
@@ -0,0 +1,35 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+# cabinet_turnover --------------------------------------------------------------------
+cabinet_turnover <- read_csv("data-raw/cabinet-turnover/cabinet-turnover.csv") %>% 
+  clean_names() %>%
+  mutate(combined = str_detect(length, "combined")) %>% 
+  mutate(length = str_replace_all(length, " combined", "")) %>% 
+  mutate(length = as.numeric(length)) %>% 
+  mutate(president = factor(president))  
+
+cabinet_turnover <- cabinet_turnover %>% 
+  mutate(start = replace(start, cabinet_turnover$appointee == "Bob Gates", "12/18/2006" )) %>%
+  mutate(start = replace(start, cabinet_turnover$appointee == "George Tenet", "7/3/95" )) %>% 
+  mutate(start = replace(start, cabinet_turnover$appointee == "Lauro Cavazos", "9/20/88" )) %>% 
+  mutate(start = replace(start, cabinet_turnover$appointee == "Dick Thornburgh", "8/15/88" )) %>%
+  mutate(start = replace(start, cabinet_turnover$appointee == "Bill Webster", "5/26/87" )) %>% 
+  mutate(start = replace(start, cabinet_turnover$appointee == "Nicholas Brady", "9/15/88" )) %>%
+  mutate(end = replace(end, cabinet_turnover$appointee == "Bob Gates", "6/30/2011" )) %>%
+  mutate(end = replace(end, cabinet_turnover$appointee == "George Tenet", "7/11/97" )) %>% 
+  mutate(end = replace(end, cabinet_turnover$appointee == "Lauro Cavazos", "12/12/1990" )) %>% 
+  mutate(end = replace(end, cabinet_turnover$appointee == "Dick Thornburgh", "8/15/1991" )) %>%
+  mutate(end = replace(end, cabinet_turnover$appointee == "Bill Webster", "8/31/1991" )) %>% 
+  mutate(end = replace(end, cabinet_turnover$appointee == "Nicholas Brady", "1/17/1993" ))
+
+cabinet_turnover <- cabinet_turnover %>% 
```

Comment:
Don't forget to combine this set of pipes to the previous one, forming a single chain.